gradebookng/tool/src/webapp/scripts/gradebook-chart.js

```diff
@@ -60,15 +60,21 @@ function renderChart(gbChartData) {
 						display: true,
 						labelString: xAxisLabel,
 						fontSize: 14,
+						fontFamily: 'Monospace',
 						fontStyle: 'bold'
 					}
 				}],
 				yAxes: [{
 					ticks: {
 						beginAtZero:true,
 						fontStyle: 'bold',
+						fontFamily: 'Monospace',
 						autoskip: true,
-						maxRotation: 0
+						maxRotation: 0,
+						// Include a space to even out the plusses and minuses
+						callback: function(value, index, values) {
```

Comment:
Does it matter that we use a generic callback vs one of the chartjs specific options https://www.chartjs.org/docs/latest/axes/#callbacks?

Any issue using `' '` vs `'&nbsp;'` ?

If not, looks good to me.

Reply:
my guess is nbsp would be escaped... ' ' works as expected here

Reply:
screenshot in the JIRA